### PR TITLE
Introduce contribution guidelines

### PR DESCRIPTION
**Pull request by PitNikola**
—

Created via [Raycast](https://www.raycast.com?ref=signatureGithub)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+Thank you for your interest in contributing to Raycast Script Commands! Here you will find simple guidelines that can help you with getting started.
+
+## Guidelines
+
+### Folder structure
```
Comment (PitNikola) [inline CONTRIBUTING.md:7]: @unnamedd, does your toolkit support nested folder structures?

Comment (unnamedd) [reply CONTRIBUTING.md:7]: Nope, I will have to work on it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Reasoning behind it: to avoid automatically including scripts that people may not be interested in. E.g. if you're using Spotify scripts, there is a lower chance you will need to access Apple Music.
+
+### File naming convention
+
+Use dash-case format for script files/directories and include proper file extension, e.g. `spotify-next-track.applescript`.
+
+### Scripts that require additional modification
+
+1. Ensure that comments include instructions on how to start using the script. E.g. you might need to provide API token, install a dependency or tweak paramters.
```
Comment (unnamedd) [inline CONTRIBUTING.md:31]:
```suggestion
1. Ensure that comments include instructions on how to start using the script. E.g. you might need to provide API token, install a dependency or tweak parameters.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+2. Add `.template.` to the file name for scripts that need modifications. Then scripts won't be automatically parsed by Raycast and people who wants to use it will need to copy the file and remove `.template.` part.
```
Comment (unnamedd) [inline CONTRIBUTING.md:32]: Does that item make sense to be here, in the `### Scripts that require additional modification` section?

To me, looks a good point to have it's own section regarding templates.

Comment (crstauf) [reply CONTRIBUTING.md:32]: @unnamedd The instruction to use `.template.` is for scripts that require setup, such as my GitHub commands, and not an actual creation of a template.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+script-commands
+|
+- media
+  |
+  - spotify
+  |
+  - apple-music
+  |
+  - youtube
+```
```
Comment (japanese-goblinn) [inline CONTRIBUTING.md:21]: What about this pr #49 ? I thought no need for subdirectories

Comment (crstauf) [reply CONTRIBUTING.md:21]: @japanese-goblinn Seems there's been a change in thought, perhaps as a temporary workaround if/until Raycast supports turning off scripts by `packageName`.

Comment (japanese-goblinn) [reply CONTRIBUTING.md:21]: @crstauf yep, seems like that. Anyway this is better than manually removing no needed scripts

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+```markdown
+script-commands
+|
+- media
+  |
+  - spotify
+  |
+  - apple-music
+  |
+  - youtube
```
Comment (crstauf) [inline CONTRIBUTING.md:20]:
```suggestion
. script-commands
└─ media
   ├─ spotify
   ├─ apple-music
   └─ youtube
```

File: CONTRIBUTING.md (hunk 3 above)
Comment (unnamedd) [reply CONTRIBUTING.md:32]: For sure @crstauf you're right! I read one thing and understood another.

**Comment by crstauf** [comment]
Does, or how does, the decision to support subdirectories impact #47?

**Comment by PitNikola** [comment]
For visibility: I've left a response in regards to `packageName` requirement here: https://github.com/raycast/script-commands/issues/47

**Pull request merged**